recipes/utm/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "utm" %}
+{% set version = "0.4.2" %}
+{% set sha256 = "a6608a67df84418fd959a79b228b90ab55b2ae877827f9c210947104c5a75d0e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
It looks like the `setup.py` doesn't use `setuptools`, you just use `python setup.py install` and drop `setuptools` from the build requirements.